circuits/bid/README.md

```diff
@@ -0,0 +1,87 @@
+## bid-circuits
+![Build Status](https://github.com/dusk-network/rusk/workflows/Continuous%20integration/badge.svg)
+[![Repository](https://img.shields.io/badge/github-bid--circuits-blueviolet?logo=github)](https://github.com/dusk-network/rusk/tree/master/circuits/bid)
+[![Documentation](https://img.shields.io/badge/docs-bid--circuits-blue?logo=rust)](https://docs.rs/bid-circuits/)
+
+### Contents
+
+This library provides the implementation of the `BidCorrectnessCircuit`
+which allows the user to construct and verify a Proof of Bid correctness.
+
+Specifically, the circuit makes sure that:
+1. The public commitment is indeed the result of: `GENERATOR * bid_value +
+GENERATOR_NUMS * bid_blinder`. 2. The bid_value relies in the range [50_000,
+250_000].
+
+#### Example
+```rust
+use rand::{Rng, thread_rng};
+use dusk_plonk::prelude::*;
+use dusk_plonk::jubjub::{GENERATOR_EXTENDED, GENERATOR_NUMS_EXTENDED};
+use bid_circuits::BidCorrectnessCircuit;
+
+// ------- Common perspective ------- //
+// Generate Public Parameters (CRS)
+let pub_params =
```

Comment:
Maybe the example should contain the actual scenario we are going to have that the CRS is derived from rusk-profile?

Reply:
I don't think it will make a difference. This will be seen in rusk/build.rs anyway. Here I would prefer to have a generic-purpose description.